src/vue-testing-library.js

```diff
@@ -117,7 +117,7 @@ const changeOrInputEventCalledDirectly = (eventValue, eventKey) =>
 
 Object.keys(dtlFireEvent).forEach(key => {
   fireEvent[key] = async (...args) => {
-    if (changeOrInputEventCalledDirectly(args[1], key)) {
+    if (process.env.WARN_EVENT_UPDATE === 'true' && changeOrInputEventCalledDirectly(args[1], key)) {
```

Comment:
To make it simpler, what about doing it in negative?
If we call the variable `VTL_SKIP_WARN_EVENT_UPDATE` (to make it similar to the other env var) we can then skip setting the default value in line 191.

Reply:
Oh yeah, sounds reasonable.